e2e/regression_guardrail/guardrail.insulin_sensitivities.spec.js

```diff
@@ -0,0 +1,65 @@
+const { loop } = require('../../src/index');
+
+describe('guardrail insulin sensitivities', () => {
+    it('cannot set 501 units', async () => {
+        await loop.Launch();
+        await loop.Configure({
+            settings: {
+                AddPumpSimulator: true,
+                InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '501' }],
```

Comment:
[nit] Since I see `loop.Launch` and `loop.Configure` as test "setup", consider breaking the actual test out separately from the `loop.Configure` code, if possible.

Reply:
yeah good plan